src/cascadia/TerminalCore/ColorFix.cpp

```diff
@@ -0,0 +1,280 @@
+#include "pch.h"
+

```

Comment:
I'd also leave a big ol' note at the top here pointing at the file from conemu that this is largely a reproduction of, so that we can go cress reference easily. Similar to the one at the top of `src\cascadia\WindowsTerminal\VirtualDesktopUtils.cpp`